performant_PoCs/focused-vision/focused_pipeline/main.py

```diff
@@ -0,0 +1,156 @@
+from pathlib import Path
+
+import depthai as dai
+from depthai_nodes.node import GatherData, ParsingNeuralNetwork, ImgDetectionsBridge
+from depthai_nodes.node.utils import generate_script_content
+
+from utils.annotation_node import AnnotationNode
+from utils.grid_eyes_annotation_node import GridEyesAnnotationNode
+from utils.arguments import initialize_argparser
+from utils.grid_layout_node import GridLayoutNode
+
+_, args = initialize_argparser()
+
+INPUT_WIDTH, INPUT_HEIGHT = 3840, 2160
+DET_MODEL = "luxonis/yunet:320x240"
+EYE_MODEL = "luxonis-ml-team/eye-detection:eye-detection-512x512"
+
+visualizer = dai.RemoteConnection(httpPort=8082)
+device = dai.Device(dai.DeviceInfo(args.device)) if args.device else dai.Device()
+platform = device.getPlatform().name
+
+frame_type = dai.ImgFrame.Type.BGR888i if platform == "RVC4" else dai.ImgFrame.Type.BGR888p
+
+if not args.fps_limit:
+    args.fps_limit = 8 if platform == "RVC2" else 30
+
+with dai.Pipeline(device) as pipeline:
+    det_desc = dai.NNModelDescription(DET_MODEL)
+    det_desc.platform = platform
+    det_archive = dai.NNArchive(dai.getModelFromZoo(det_desc))
+
+    det_w = det_archive.getInputWidth()
+    det_h = det_archive.getInputHeight()
+
+    eye_model_description = dai.NNModelDescription(EYE_MODEL)
```

Comment:
Use consistent variable naming. For the head detection model description, you used `det_desc`, now for the same thing but different model you use `eye_model_description` -> this is inconsistent naming. Same things have to be named the same